src/components/FeaturedSnippetDirectAnswer.tsx

```diff
@@ -29,17 +30,32 @@ interface FeaturedSnippetDirectAnswerCssClasses {
  *
  * @internal
  */
+
+const unsupportedTextFormats: string[] = ['rich_text', 'rich_text_v2', 'markdown'];
+
 export function FeaturedSnippetDirectAnswer({
   result,
   readMoreClickHandler,
   cssClasses = {}
 }: FeaturedSnippetDirectAnswerProps): JSX.Element {
   const answer = result.fieldType === 'multi_line_text' && result.value;
-  // TODO: SLAP-2340, update rich text snippets to convert the markdown
-  if (result.fieldType === 'rich_text') {
-    console.warn('Rendering markdown for rich text direct answer is currently not supported. Displaying the unrendered markdown string as is.');
+  if (unsupportedTextFormats.includes(result.fieldType)) {
+    console.warn('Rendering ' + result.fieldType + ' direct answer is currently not supported. ' +
+        'You can modify your search configuration to convert ' + result.fieldType + ' to HTML to be rendered '
+        + 'on the page.');
+  }
+  let snippet: JSX.Element;
+  const snippetValue = useMemo(() =>
+  { return { __html: result.snippet.value }; }, [result.snippet.value]);
```

Comment:
can we update these to be `result.snippet?.value`? That way if the snippet isn't a property of the result, the code won't crash by trying to access `snippet.value`